src/components/Book/BookItem.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] It's not a good approach to disable linters, fixing linters help us to write a clean, clear, and good codebase, kindly make sure to not disable linters 🙏🏾